web/home_test.go

```diff
@@ -1,20 +1,41 @@
 package web
 
 import (
+	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
+	"strings"
 	"testing"
-
-	"github.com/stretchr/testify/assert"
 )
 
 func Test_homeHandler(t *testing.T) {
-	engine := NewEngine()
 
-	resp := httptest.NewRecorder()
-	req, _ := http.NewRequest("GET", "/", nil)
-	engine.ServeHTTP(resp, req)
+	templs := AddAllTemplatesFromFS(templatesFS, "templates/*.tmpl")
+	req, err := http.NewRequest("GET", "/", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	rr := httptest.NewRecorder()
+	handler := http.HandlerFunc(IndexHandler(templs))
+	handler.ServeHTTP(rr, req)
+
+	// Check the status code is what we expect.
+	if status := rr.Code; status != http.StatusOK {
+		t.Errorf("handler returned wrong status code: got %v want %v",
+			status, http.StatusOK)
+	}
+
+	// test body with console expected text
+	expected := "SUSE Console for SAP Applications"
+	if b, err := ioutil.ReadAll(rr.Body); err != nil {
+		t.Fail()
+	} else {
+		if strings.Contains(string(b), "Error") {
+			t.Errorf("header response shouldn't return error: %s", b)
+		} else if !strings.Contains(string(b), expected) {
+			t.Errorf("header response doesn't match:\n%s", b)
+		}
+	}
```

Comment:
I don't really understand why did you change this test, I don't think this should be needed at all. The test used the default http testing toolkit and `testify` assertions, could you please reinstate that?
I think you're only need to invoke `ServeHTTP` on a different struct.

Reply:
this is more subjective taste on how to test, but I personally don't care to reintroduce what was before